### DISCUSSION TITLE
Does FiveServer also work with DDEV?

### DISCUSSION THREAD

**Discussion by slowcoder76** [yandeu/five-server#83] [Help / Questions] [answered]
When I use php: "/opt/hombebrew/bin/php" on MacOS, simple php pages working. But .htaccess is not taken into account.
Does FiveServer also work with DDEV?

Best Regards SlowCoder

</Translated with GoogleTranslator>

**Comment by yandeu** [comment, marked as answer]
Yes .htaccess does not work since five server does not provide a full PHP Server.

I have never used DDEV.

Please have a loot at this example:  
https://github.com/yandeu/five-server-with-php

**Reply by slowcoder76** [reply]
Thank you for your quick reply.  Then I guess I just need a LiveReload plugin.

**Reply by yandeu** [reply]
If DDEV offers a LiveServer Plugin, then yes. Else, you could use five-server (npm version) instead.
